src/graphql/node.rs

```diff
@@ -462,23 +462,23 @@ impl NodeStatus {
     async fn id(&self) -> ID {
         ID(self.id.to_string())
     }
-
+    /// The RAM size in bytes. The type must be u64.
     async fn total_memory(&self) -> Option<StringNumber<u64>> {
         self.total_memory.map(StringNumber)
     }
-
+    /// The amount of used RAM in bytes. The type must be u64.
```

Comment:
This is the same as what you wrote in the first place. I think it is a minor mistake, but these are the kinds of errors that waste our time.

Reply:
I am really sorry. There were some critical mistakes when I was controlling rebase and reset commits through git command. I should have checked the diff more meticulously.